samples/Actor/IDemoActor/IDemoActor.csproj

```diff
@@ -5,7 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
```

Comment:
This can be removed as its not used

Reply:
The dependency of Newtonsoft.Json is derived from the original code, of course I can remove it.